app/assets/stylesheets/views/_choose_sign_in.scss

```diff
@@ -0,0 +1,22 @@
+.verify-hint-box {
+  width: 100%;
+  border: 5px solid $govuk-border-colour;
+  padding: 15px;
+  box-sizing: border-box;
+  margin: -30px 0 30px;
```

Comment:
As Ian mentioned, can we use the spacing scale at least for vertical spacing (e.g.  `- govuk-spacing(6) 0 govuk-spacing(6)`)